_search-plugins/knn/knn-index.md

```diff
@@ -112,9 +114,41 @@ The IVF algorithm requires a training step. To create an index that uses IVF, yo
 data points, but it is [recommended to use more](https://github.com/facebookresearch/faiss/wiki/Guidelines-to-choose-an-index#how-big-is-the-dataset).
 Training data can either the same data that is going to be ingested or a separate set of data.
```

Comment:
We need a verb either before or after "either". Can we change "set of data" to "dataset"?

Reply:
"Training data can be composed of either the same data that is going to be ingested or a separate dataset."